src/main/resources/openapi/schemas/ingestJobInfo.json

```diff
@@ -0,0 +1,14 @@
+{
+  "description": "Ingest job info",
+  "type": "object",
+  "properties": {
+    "autoRollBackTimeout": {
+      "type": "integer",
+      "description": "Timeout in seconds"
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "sourceId"
```

Comment:
sourceId is required but not defined in the "properties" section

Reply:
Roger.. No required properties then  - at this stage.